Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/RiskIQDigitalFootprint.py

```diff
@@ -0,0 +1,2451 @@
+        if parent_key in sub_key:
+            pascal_key = sub_key
+        flat_dict[pascal_key] = sub_value
+
+    return flat_dict
+
+
+def prepare_context_dict(response_dict: Dict[str, Any],
+                         keys_with_hierarchy: Union[tuple, str] = (),
+                         exclude_keys: Union[tuple, str] = ()) -> Dict[str, str]:
+    """
+    Prepare the context dictionary as per the standards.
+
+    :param response_dict: dictionary getting from API response that contains sub-dictionaries
+    :param keys_with_hierarchy: list of keys that contains sub-dictionary as its value.
+    :param exclude_keys: keys need to exclude.
+    :return: single level dictionary
+    """
+    simple_dict: Dict[str, str] = {}
+    for key, value in response_dict.items():
+        if key in keys_with_hierarchy:
+            simple_dict.update(nested_to_flat(response_dict.get(key, {}), key))
+        elif key not in exclude_keys:
+            simple_dict[key] = value
+    return simple_dict
+
+
+def validate_asset_connections_args(type_arg: str, global_arg: str) -> Tuple[str, Any]:
+    """
+    Validate the arguments passed by the user
+
+    :param type_arg: asset_type
+    :param global_arg: argument which defines if connected assets are to be fetched from global inventory
+    :return: type_arg, global_arg: asset_type and global arguments passed by user or message describing error that
+     occurred in parameter validation
+     occurred during validation
+    """
+    if type_arg not in VALID_ASSET_TYPES:
+        raise ValueError(MESSAGES['INVALID_ASSET_TYPE_ERROR'])
+    if global_arg:
+        global_arg = global_arg.lower()
+        try:
+            global_arg = argToBoolean(global_arg)
+        except ValueError:
+            raise ValueError(MESSAGES['INVALID_BOOLEAN_VALUE_ERROR'].format('global'))
+
+    if type_arg == 'ASN':
+        type_arg = 'AS'
+
+    return type_arg, global_arg
+
+
+def get_comma_separated_values(values: List[Dict[str, Any]], key: str) -> str:
+    """
+    Retrieve values of an attribute in comma separated manner from response to populate human readable output
+
+    :param: values: list of attributes fetched from response
+    :param: key: key to consider the value for
+    :return: a string containing values of passed attribute in comma separated manner
+    """
+    value_list = set()
+    for value in values:
+        value_list.add(value.get(key, ''))
+    return ', '.join(value_list)
+
+
+def get_comma_separated_values_only_current(values: List[Dict[str, Any]], key: str) -> str:
+    """
+    Retrieve values of an attribute in comma separated manner from response to populate human readable output
+
+    :param: values: list of attributes fetched from response
+    :param: key: the key to fetch the value of
+    :return: a string containing values of passed attribute in comma separated manner
+    """
+    value_list = set()
+    for value in values:
+        if value.get('current', None):
```

Comment:
```suggestion
        if value.get('current'):
```
The value returned by a dictionary's `.get()` method if the passed key is not found, is by default `None`. You only need to specify a default value if you'd like it to be something other than `None`.

Reply:
@avidan-H We will make this change.